src/translations.cpp

```diff
@@ -68,7 +68,7 @@ void set_language(bool reload_options)
 
     // Step 2. Bind to gettext domain.
     const char *locale_dir;
-#ifdef __linux__
```

Comment:
I think further that this conditional depending on the platform type such as `__linux__` is fragile. For example, if one wants to build a localized curses version on FreeBSD, one suffers from the same problem as I did. So one needs to extend this conditional by adding `__FreeBSD__`. By the way, there are many UNIX-like platforms other than Linux and FreeBSD.